docs/using-avail/use-cases-and-examples/validiums.md

```diff
@@ -0,0 +1,42 @@
+---
+id: use-case-validiums
+title: Avail-Powered Validiums
+sidebar_label: Validiums
+description: How to use Avail to build validiums
+keywords:
+  - docs
+  - avail
+  - availability
+  - scale
+  - rollup
+image: https://availproject.github.io/img/avail/AvailDocs.png
+slug: /using-avail/validiums
+---
+
+# Avail-Powered Validiums
+
+Due to the architecture of monolithic blockchains (such as Ethereum in
+its current state), operating the blockchain is expensive, resulting
+in high transaction fees. Rollups attempt to extract the burden of
+execution by running transactions off-chain and then posting the
+execution results and the [usually compressed] transaction data.
+
+Validiums are the next step: instead of posting the transaction data,
+it is kept available off-chain, where a proof/attestation is only
+posted to the base layer. This is by far the most cost-effective
+solution because both execution and data availability are kept
+off-chain while still allowing for final verification and settlement
+on the layer 1 chain.
+
+Avail is a blockchain optimized for data availability. Any rollup that
+wishes to become a validium can switch to post transaction data to
+Avail instead of the layer 1 and deploy a verification contract that,
+in addition to verifying the correct execution, also verifies data
+availability.
+
+The Avail team will make this data availability verification simple on
```

Comment:
@0xSasaPrsic Can you please help us explain use of validium in this page (or in a subpage)? In particular, we would want to document some of the details of using our attestation bridge here. 
@vthunder thoughts?

Reply:
yes, I will add example of using attestation bridge.

Reply:
Note that I just moved this page over from the old docs. Feel free to modify it as needed.